pkg/types/100/types_test.go

```diff
@@ -321,4 +326,54 @@ var _ = Describe("Current types operations", func() {
     "address": "10.1.2.3/24"
 }`))
 	})
+
+	Context("when setting custom routes with mtu 0", func() {
```

Comment:
We should have a test for when mtu is actually set as well.  e.g. set to e.g. 1400 (adjust variable names to taste.)

```
+
+       Context("when setting custom routes with mtu 1400", func() {
+               notzero := 1400
+
+               exampleMTU := current.Route{
+                       Dst: net.IPNet{
+                               IP:   net.ParseIP("1.2.3.0"),
+                               Mask: net.CIDRMask(24, 32),
+                       },
+                       GW:  net.ParseIP("1.2.3.1"),
+                       MTU: &notzero,
+               }
+
+               It("marshals to JSON equivalent to an omitted mtu ", func() {
+                       jsonBytes, err := json.Marshal(&exampleMTU)
+                       Expect(err).NotTo(HaveOccurred())
+                       Expect(jsonBytes).To(MatchJSON(`{ "dst": "1.2.3.0/24", "gw": "1.2.3.1", "mtu": 1400 }`))
+               })
+       })

```

Reply:
There were other tests that had covered this path (`testResult`), but I've added it to make the tests in this function complete.